.travis.yml

```diff
@@ -11,7 +11,6 @@ env:
   - DJANGO_VERSION=1.9
   - DJANGO_VERSION=1.10
   - DJANGO_VERSION=1.11
-  - DJANGO_VERSION=2.0
```

Comment:
You should keep this line but exclude the Django 2.0 / Python 2.7 test:
https://docs.travis-ci.com/user/customizing-the-build/#excluding-jobs